### PR TITLE
Rewrite of release schedule FAQ

### PR DESCRIPTION
**Pull request by SeanPM5**
## Proposed change

Rewrote everything to be more concise and easier to read. Removed two sentences that I felt were unnecessary (how time zones work).

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_faq/release.markdown [hunk 1]
```diff
@@ -1,12 +1,9 @@
 ---
 title: "Releases"
-description: "Releases"
+description: "Home Assistant release schedule"
 ha_category: Common
 ---
 
-The current release cycle is three weeks. This allows for two weeks of development time and 1 week of beta testing and fixing. Releases usually occur on a Wednesday. There is no fixed time of day when the release will happen because that depends on the person who is finishing the release, it could therefore be Tuesday or Thursday depending on your timezone when its available for you. If there was no announcement made in the previous release notes or on another communication channel then the release will happen.
+New major versions of Home Assistant are released every three weeks on a Wednesday. The exact dates can be seen in the upcoming events calendar on the [Home Assistant Developers](https://developers.home-assistant.io/) website.
```
Comment (frenck) [inline source/_faq/release.markdown]: Actually, minor, as we have not reached 1.0 yet.

File: source/_faq/release.markdown [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
+New major versions of Home Assistant are released every three weeks on a Wednesday. The exact dates can be seen in the upcoming events calendar on the [Home Assistant Developers](https://developers.home-assistant.io/) website. 
 
-Beta's are normally released on the 2nd Wednesday after the last major release. Please subscribe to the Beta Channel on the [Home Assistant Discord server](https://discord.gg/RMC8WkX) for help running beta versions.
-When a beta release is in progress, the release notes for it can be found under [Beta Release Notes](https://rc.home-assistant.io/latest-release-notes/).
-
-The release schedule is published in the upcoming events calender on the [Home Assistant Developers](https://developers.home-assistant.io/) website.
+This release schedule allows for two weeks of development time and one week of beta testing. Users who participate in the beta can view the changelog under [beta release notes](https://rc.home-assistant.io/latest-release-notes/) and get help in the `#beta` channel of [Home Assistant's Discord server](https://discord.gg/RMC8WkX). Testers are also encouraged to [report issues on GitHub](/help/#bugs-feature-requests-and-alike).
```
Comment (frenck) [inline source/_faq/release.markdown]: This is not correct. We have a 3 week development cycle. With the release of the beta, the development of the next release starts as well.

Comment (frenck) [reply source/_faq/release.markdown]: use `/join-chat` for linking to our Discord.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @SeanPM5!

Left a couple of small comments, could you take a look? 👍

File: source/_faq/release.markdown (hunk 1 above)
Comment (SeanPM5) [reply source/_faq/release.markdown]:
```suggestion
New versions of Home Assistant are released every three weeks on a Wednesday. The exact dates can be seen in the upcoming events calendar on the [Home Assistant Developers](https://developers.home-assistant.io/) website. 
```
My intention with "major" was to try and differentiate between feature releases and hotfixes. But in hindsight that doesn't belong here. I'll remove the word completely.

File: source/_faq/release.markdown (hunk 2 above)
Comment (SeanPM5) [reply source/_faq/release.markdown]:
```suggestion
The third week of our release schedule is primarily focused on beta testing. Users who participate in the beta can view the changelog under [beta release notes](https://rc.home-assistant.io/latest-release-notes/) and get help in the `#beta` channel of [Home Assistant's Discord server](/join-chat). Testers are also encouraged to [report issues on GitHub](/help/#bugs-feature-requests-and-alike).
```
Maybe this is better?

**Review by frenck — APPROVED** [review]
Thanks, @SeanPM5 👍 

Happy Hacktoberfest 🎉

**Pull request merged**